gatsby-config.js

```diff
@@ -3,6 +3,7 @@ module.exports = {
     title: "Cloudflare Docs",
     description: "Documentation for Cloudflare products and services.",
     author: "@cloudflare",
+    siteUrl: "http://workers.cloudflaredocs.workers.dev",
```

Comment:
@adamschwartz we'll have to change this when we move the docs, it's used as the base URL in the sitemap

Reply:
Maybe add a `// TODO: update with real URL` comment?